server.go

```diff
@@ -342,7 +343,22 @@ func CustomCodec(codec Codec) ServerOption {
 // later release.
 func ForceServerCodec(codec encoding.Codec) ServerOption {
 	return newFuncServerOption(func(o *serverOptions) {
-		o.codec = codec
+		o.codec = newCodecV1Bridge(codec)
+	})
+}
+
+// ForceServerCodecV2 is the equivalent of ForceServerCodec, but for the new
+// CodecV2 interface.
+//
+// Will be supported throughout 1.x.
```

Comment:
We generally add this line when we deprecate APIs I thought.

Reply:
Yeah I should remove that

Reply:
There are other functions in this file that have the same comment (for example [this one](https://pkg.go.dev/google.golang.org/grpc#ForceServerCodec))